services/src/pro/users/postgres_userdb.rs

```diff
@@ -436,7 +451,13 @@ where
     async fn increment_quota_used(&self, user: &UserId, quota_used: u64) -> Result<()> {
         let conn = self.conn_pool.get().await?;
         let stmt = conn
-            .prepare("UPDATE users SET quota_used = quota_used + $1 WHERE id = $2;")
+            .prepare(
+                "
+            UPDATE users SET 
+                quota_available = quota_available - $1, 
```

Comment:
add a test where the quota_available gets negative after a query

Reply:
bf8442899e6d3725431a335c3cf664c008a11173